Metadata.Rmd

```diff
@@ -0,0 +1,660 @@
+# Package metadata {#description}
+
+```{r, include = FALSE}
+source("common.R")
+```
+
+The job of the `DESCRIPTION` file is to store important metadata about your package.
+When you first start writing packages, you'll mostly use these metadata to record what packages are needed to run your package.
+However, as time goes by and you start sharing your package with others, the metadata file becomes increasingly important because it specifies who can use it (the license) and whom to contact (you!) if there are any problems.
+
+Every package must have a `DESCRIPTION`.
+In fact, it's the defining feature of a package (RStudio and devtools consider any directory containing `DESCRIPTION` to be a package)[^DESC-and-package-hood].
+To get you started, `usethis::create_package("mypackage")` automatically adds a bare-bones `DESCRIPTION` file.
+This will allow you to start writing the package without having to worry about the metadata until you need to.
+The minimal description will vary a bit depending on your settings, but should look something like this:
+
+[^DESC-and-package-hood]:
+The relationship between "has a `DESCRIPTION` file" and "is a package" is not quite this clear-cut.
```

Comment:
OK, I guess the book is technically some sort of edge case of a package. My goal here is mostly just to point out the usefulness of `DESCRIPTION` in, shall we say, projects that don't follow many or any other package conventions.

Also, the IDE definitely recognizes the R Packages project as a bookdown project, not a package, in terms of how the Build Pane looks, which keyboard shortcuts are "live", etc.

Reply:
It might be worthing mentioning that (I think) what Build pane RStudio shows is determined by the `BuildType:` field in the `.Rproj` file.; I think the choices are one of `None`/`Package`/`Makefile`/`Website`/`Custom` - and the Build pane doesn't appear for `None`.